OMGVA PoS/Data layer/Repositories/Tax/ITaxRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace OMGVA_PoS.Data_layer.Repositories.Tax;
+
+public interface ITaxRepository
+{
+    public Task<List<Models.Tax>> GetAllTaxesAsync();
```

Comment:
Why not create an alias for the Models namespace? Then you could use Tax instead of Models.Tax.
This can be done by adding this before the namespace:
`using OMGVA_PoS.Data_layer.Models;`